docs/getting_started.md

```diff
@@ -85,7 +85,7 @@ helm delete my-datadog-operator
 [2]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
 [3]: https://artifacthub.io/packages/helm/datadog/datadog-operator
 [4]: https://app.datadoghq.com/account/settings#api
-[5]: https://github.com/DataDog/datadog-operator/blob/main/docs/configuration.md
+[5]: https://github.com/DataDog/datadog-operator/blob/main/docs/configuration.v1alpha1.md
```

Comment:
How did you decide that this should be v1 versus v2?

Reply:
The embedded YAML above refers to `v1alpha1` on line 66, so I figured it made more sense to link to that version.

Reply:
IMHO, the docs are in an awkward state where `v2` seems to be the latest and greatest, but most of the examples still refer to `v1`. I dunno whether you want to change all the examples over to v2, or have parallel sets of instructions and examples for v1 and v2, or some other solution.

I wanted to keep the changes in my PR to be minimal, and just fix a few things that were clearly broken or incorrect.